src/client/providers/settings/settings.ts

```diff
@@ -0,0 +1,8 @@
+import { UserSettings } from "../../types.js";
+
+export const settings: UserSettings = {
+    userId: 1,
+    settings: {
+        toolApprovalRequired: true,
+    },
+}
```

Comment:
let's rename this file to `src/settings/utils.ts` and fetch these settings from a function `getDefaultUserSettings`